http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/MonitorAtBoot.scala

```diff
@@ -37,7 +37,7 @@ class MonitorAtBoot[F[_]](publisherQueue: Queue[F, LeoPubsubMessage],
         for {
           now <- F.realTimeInstant
           implicit0(traceId: Ask[F, TraceId]) = Ask.const[F, TraceId](TraceId(s"BootMonitor${now}"))
-          _ <- handleRuntime(r)
+          _ <- handleRuntime(r, None) // Should we pass in the custom timeout here, and if so how?
```

Comment:
@Qi77Qi FYI this is a weird blocker that I am unable to answer myself. How should I pass the timeout argument in this function?

Reply:
let's leave it out for now...this class is for recovering cases when runtimes are left in transit status after Leo is restarted.